chia/wallet/wallet_node.py

```diff
@@ -536,14 +536,14 @@ def is_new_state_update(cs: CoinState) -> bool:
 
         # The number of coin id updates are usually going to be significantly less than ph updates, so we can
         # sync from 0 every time.
-        continue_while = False
+        continue_while = True
         all_coin_ids: List[bytes32] = await self.get_coin_ids_to_subscribe(0)
         already_checked_coin_ids: Set[bytes32] = set()
         while continue_while:
             one_k_chunks = chunks(all_coin_ids, 1000)
             for chunk in one_k_chunks:
                 c_update_res: List[CoinState] = await subscribe_to_coin_updates(chunk, full_node, 0)
-                c_update_res = list(filter(is_new_state_update, c_update_res))
```

Comment:
That's a pretty major change here - totally fine, as long as the downstream code `self.receive_state_from_peer` is prepared for it.

BTW, is
    `ph_update_res = list(filter(is_new_state_update, ph_update_res))`
above still valid as well?

Reply:
Yeah we want to filter PH stuff but not coin stuff, the reason is that there are many many more PH updates usually, coin subscriptions are only really needed for a few things like plotnfts, so it's fine to not filter them. Worst case we reprocess some stuff. 

Before this PR, this code wasn't even being ran due to the `continue_while` value being False.